src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,115 @@
+package duke.parser;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.Operation;
+import duke.command.SearchCommand;
+import duke.exception.DukeException;
+import duke.exception.EmptyInputException;
+import duke.exception.NotRecognizeException;
+import duke.task.Task;
+
+public class Parser {
+    /**
+     * Returns boolean value which checks whether
+     * input string is digit or not.
+     *
+     * @param input String input.
+     * @return Content of input is digit or not.
+     */
+    public static boolean chekDigit(String input) {
```

Comment:
should checkDigit be spelled like that?

Reply:
oops, i haven't detected it before, nice catch!